### ISSUE TITLE
alt-up/down doesn't do anything, and ctrl-shift-a is confusing

### ISSUE THREAD

**Issue by izuzak** [atom/atom#1870]
Halp ticket:
- support/d8c483c6c4af11e3900abd3beebdf506

Pressing `alt-up` and `alt-down` doesn't move the cursor at all, which isn't what some of our users expect. The same is true for `alt-shift-up` and `alt-shift-down`.

As far as I can tell, SublimeText3 uses `alt-up/down` to jump to the beginning/end of the line, and `shift-alt-up/down` to select to the beginning/end of the line. Might this be a good default in Atom as well?

This behavior of jumping to the beginning of the line is currently bound to `ctrl-a` (`editor:move-to-beginning-of-line`), while `cmd-left` jumps to the beginning of the physical line (`editor:move-to-first-character-of-line`).

However, `ctrl-shift-a` doesn't select to the beginning of the logical line but to the beginning of the physical line (`editor:select-to-first-character-of-line`), which is the same thing as `cmd-shift-left`.

In other words, if `ctrl-shift-a` was consistent with `ctrl-a`, it would select like this:

![screen shot 2014-04-15 at 7 33 13 pm](https://cloud.githubusercontent.com/assets/38924/2710664/4e4bfff2-c4c4-11e3-8bfa-78748503f9aa.png)

while it currently selects like this:

![screen shot 2014-04-15 at 7 33 30 pm](https://cloud.githubusercontent.com/assets/38924/2710666/53c7b8c2-c4c4-11e3-8ddc-5d961800e834.png)

**Comment by cacheflow** [comment]
Thanks for recommending this.
